README.md

```diff
@@ -18,30 +16,26 @@ The root contract of a Plasma child chain represents an intermediary who can res
 ## Root Contract Details
 A transaction is encoded in the following form:
 
-``[Blknum1, TxIndex1, Oindex1, Amount1, ConfirmSig1,``
-
-``Blknum2, TxIndex2, Oindex2, Amount2, ConfirmSig2,``
-
-``NewOwner, Denom1, NewOwner, Denom2, Fee]``
-
-
-``submitBlock``: Validator(s) submits merkle root of the current block
-
-``deposit``: Users will deposit onto the child chain by calling ``deposit()``. Deposits are stored into a priority queue and processed upon the next call of ``submitBlock()``. **Note**: If a validator decides to never call submitBlock again, users with pending deposits will still be able to withdraw their deposit
-
-``startExit``: When a user decides to exit for any reason, they will call ``startExit()`` and their pending exit will be added to a priority queue that has a 1 week challenge period. This function requires a bond.
-
-``challengeExit``: If any users notice an invalid withdrawal attempt, they may challenge this exit by providing the relevant information. If their challenge is successful, they will be awarded the bond associated with the exit attempt.
+```
+[Blknum1, TxIndex1, Oindex1, DepositNonce1, Amount1, ConfirmSig1
```

Comment:
Tx also includes Owner1 and Owner2

Reply:
Should we make that apart of the compatibility PR

Reply:
Yes definitely, I did not realize this pr would have no compatibility changes